apps/website/src/components/documentation/tsdoc/TSDoc.tsx

```diff
@@ -84,6 +84,12 @@ export function TSDoc({ item, tsdoc }: { readonly item: ApiItem; readonly tsdoc:
 								itemURI={resolveItemURI(foundItem ?? resolved!.item)}
 								key={idx}
 								packageName={resolved?.package ?? item.getAssociatedPackage()?.displayName.replace('@discordjs/', '')}
+								version={
+									resolved?.package
+										? // eslint-disable-next-line unicorn/better-regex
+										  item.getAssociatedPackage()?.dependencies?.[resolved.package]?.replace(/[~^]/, '')
```

Comment:
`item.getAssociatedPackage()` is called 3 times in this block, do you think it is necessary to cache the result of this?

<details>
<summary>Example</summary>

```ts
						let associatedPackage: ApiPackage | null | undefined = null;

						// eslint-disable-next-line no-inner-declarations
						function getAssociatedPackage() {
							if (associatedPackage !== null) return associatedPackage;
							return (associatedPackage = item.getAssociatedPackage());
						}
```
</details>

Reply:
Might be a good idea. But if we talk about caching stuff it would be way better to cache the ApiModel/ApiPackage in a more central place, because currently the website rebuilds the Model a lot...